UI/Pages/Shared/_OptionsSelect.cshtml

```diff
@@ -3,10 +3,7 @@
 <div class="app-c-option-select js-collapsible" data-testid="@Model.Name" data-module="option-select" @Model.ClosedData>
 
     <div class="app-c-option-select__heading js-container-heading">
-        <button class="js-container-button app-c-option-select__title app-c-option-select__button" type="button"
-                id="option-select-title-@Model.Name" aria-controls="list-of-@Model.Name">
-            @Model.DisplayName
-        </button>
+        <span class="app-c-option-select__title js-container-button" id="option-select-title-@Model.Name">@Model.DisplayName</span>
```

Comment:
Could we use the existing `<legend>` from the `<fieldset>` for no-JS so that it looks the same as the 'Type of tuition'? Or move the existing legend to here, and set it as visually hidden is JS is enabled maybe?

Reply:
Made it so that it shows the button if js is enabled, but shows legend (updated to "govuk-fieldset__legend--s") if not enabled.